rpskills/CoreSystem/HeritageSelectionNet.cs

```diff
@@ -0,0 +1,19 @@
+
+
+namespace rpskills.CoreSys
+{
+    class HeritageSelectedState
+    {
+        public bool DidSelect;
+    }
+
+    /// <summary>
+    /// Contains all data regarding Heritage selection.
+    /// 
+    /// See Vintagestory.GameContent.CharacterSelectionPacket for more details.
+    /// </summary>
+    class HeritageSelectionPacket
```

Comment:
Should include an an a way to lookup the heritage either using an ID based system checking the heritage ID of the client selection on the server or using the same VS Survival route of using a string and doing it by name.